cmd/serverless-init/cloudservice/containerapp.go

```diff
@@ -48,24 +48,29 @@ func (c *ContainerApp) GetTags() map[string]string {
 	replica := os.Getenv(ContainerAppReplicaName)
 
 	tags := map[string]string{
-		"app_name":     appName,
-		"region":       region,
-		"revision":     revision,
-		"replica_name": replica,
-		"origin":       c.GetOrigin(),
-		"_dd.origin":   c.GetOrigin(),
+		"app_name":         appName,
+		"region":           region,
+		"revision":         revision,
+		"replica_name":     replica,
+		"aca.replica.name": replica,
```

Comment:
Do you want all of these new `aca` tags to be shown in the flame graph UI? If not it might be nice to prefix them with either `_dd` or just `_` (this also causes them to be skipped for some types of obfuscation which can be nice)

Reply:
Even though this change was motivated by billing I do think that these are helpful tags to have available for customers to use to filter to spans related to a specific resource/subscription/replica.

Reply:
Sounds good, although it may be a bit confusing (and inefficient) to have duplicates here, is there a plan to deprecate and remove the "old" names?

Reply:
It's a good point because I agree it is messy having these duplicate tags. I chatted with my team internally and there is not a plan to deprecate them at this time since we want to avoid any breaking changes for customers using the "old" tags. That's not to say we will never deprecate them but there would need to be more reasons to warrant a new major version than just removing these duplicate tags.